docs/assets/highlight.css

```diff
@@ -1,22 +0,0 @@
-:root {
```

Comment:
did you move or remove the docs? Can't find them in the repo

Reply:
Added a separate issue for that. DXJ-217